packages/thumbprint-react/components/Image/components/aspect-ratio/index.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+const AspectRatio = ({ children, ratio }) => (
```

Comment:
Any reason not to add the aspect ratio calc to the `<picture>`? Would make the HTML a little cleaner. https://tomgenoni.github.io/experiments/aspect-ratio/